gather-scheduler.js

```diff
@@ -90,10 +91,11 @@ class GatherScheduler {
       .then(traceContents => {
         tracingData.traceContents = traceContents;
       })
+      .then(_ => this.saveAssets(saveTrace, tracingData, url))
```

Comment:
well, now that you say that, if our goal is to be nice and readable, the explicit `if (saveTrace) {...` may be better :)